src/dialogs/DontAskAgainDialog/DontAskAgainDialog.js

```diff
@@ -0,0 +1,153 @@
+        <FormControlLabel
+          control={
+            <Checkbox
+              data-cy={id + '-dontAskAgain-checkbox'}
+              checked={isDontAskAgainChecked}
+              onChange={handleChangeAsk}
+              id="dontAskAgainChecked"
+              color="primary"
+            />
+          }
+          label={labels.checkboxDontAskAgain}
+        />
+      </DialogContent>
+      <DialogActions>
+        <Button
+          autoFocus
+          data-cy={id + '-cancel-button'}
+          id={id + '-cancel-button'}
+          onClick={handleCloseDialog}
+          color="primary"
+        >
+          {labels.buttonCancel}
+        </Button>
+        <Button
+          data-cy={id + '-confirm-button'}
+          id={id + '-confirm-button'}
+          onClick={onButtonConfirmClick}
+          color="primary"
+        >
+          {labels.buttonConfirm}
+        </Button>
+      </DialogActions>
+    </Dialog>
+  );
+};
+
+DontAskAgainDialog.propTypes = {
+  /**
+   * Dialog's id
+   */
+  id: PropTypes.string.isRequired,
+  /**
+   *  ## Define the DoNoAskAgainDialog's state:
+   *  - true : the dialog is opened
+   *  - false : the dialog is closed
+   */
+  open: PropTypes.bool,
+  /**
+   * ## Function used to close the dialog
+   */
+  closeDialog: PropTypes.func.isRequired,
+  /**
+   *  ## Labels for dialog
+   * Structure:
+   * <pre>
+   * {
+   *  title:'This is a title',
+   *  body:'This is a body',
+   *  buttonCancel: 'Cancel',
+   *  buttonConfirm: 'Confirm',
+   *  ariaLabelledby: 'Wonderful dialog
+   *  checkboxDontAskAgain: 'Do not ask again"
+   *}
+   *</pre>
+   */
+  labels: PropTypes.shape({
+    title: PropTypes.string.isRequired,
+    body: PropTypes.string.isRequired,
+    buttonCancel: PropTypes.string.isRequired,
+    buttonConfirm: PropTypes.string.isRequired,
+    checkboxDontAskAgain: PropTypes.string.isRequired,
+    ariaLabelledby: PropTypes.string.isRequired,
+  }),
+  /**
+   * ## Function used when do no ask again checkox is cheked
+   */
+  setDontAskAgainToConfirmLaunch: PropTypes.func,
+
+  /**
+   *  ## Function used when buttonConfirm is clicked
+   */
+  handleClickOnButtonConfirm: PropTypes.func.isRequired,
+};
+
+DontAskAgainDialog.defaultProps = {
+  open: false,
+  setDontAskAgainToConfirmLaunch: null,
+  labels: {
+    title: 'Do not ask again dialog',
+    body: `
+              Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+              Vivamus quis efficitur odio, sit amet auctor nunc.
+              Curabitur et ante a nulla dapibus ultricies.
+              Proin porttitor tempor libero euismod convallis.
+              Etiam quis sollicitudin mauris. Curabitur hendrerit felis quis ligula volutpat posuere.
+              Sed gravida arcu in porttitor ornare.
+              Nulla ac rhoncus dui.
+          `,
+    buttonCancel: 'Cancel',
+    buttonConfirm: 'Confirm',
+    checkboxDontAskAgain: 'Do not ask again',
```

Comment:
Can we rename these to `cancel`, `confirm` and `doNotAskAgain` ? We probably don't need the "button" and" checkbox" info in the labels if there is only one label for each type in the dialog.